Source/JavaScriptCore/runtime/IntlDateTimeFormat.cpp

```diff
@@ -102,15 +102,13 @@ void IntlDateTimeFormat::setBoundFormat(VM& vm, JSBoundFunction* format)
     m_boundFormat.set(vm, this, format);
 }
 
-static String canonicalizeTimeZoneName(const String& timeZoneName)
+// https://tc39.es/ecma402/#sec-getavailablenamedtimezoneidentifier
+static String getAvailableNamedTimeZoneIdentifier(StringView timeZoneName)
```

Comment:
WebKit coding style reserves “get” for only certain kinds of function names, so even though the standard calls this `GetAvailableNamedTimeZoneIdentifier` we should probably name this `availableNamedTimeZoneIdentifier`.

Reply:
Fixed